swagger-spec/schemas/definition.yaml

```diff
@@ -0,0 +1,103 @@
+type: object
```

Comment:
seems like `/schemas/definition.yaml` should be the definition of the schema type, and the schema-block definition should be in `/schemas/schema_block_definition.yaml` (or something like that -- just following what seems to be a convention)